src/Microsoft.Health.Dicom.SchemaManager.Console/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "Microsoft.Health.Dicom.SchemaManager": {
       "commandName": "Project",
-      "commandLineArgs": "apply --connection-string \"server=(local);Initial Catalog=DICOM3;TrustServerCertificate=True;Integrated Security=True\" --version 20",
+      "commandLineArgs": "apply --connection-string \"server=(local);Initial Catalog=DICOM3;TrustServerCertificate=True;Integrated Security=True\" --version 20 -wi=false",
```

Comment:
I would have liked to use a switch argument, but this sort of argument works better with our usage in PaaS (and the underlying command line config source doesn't work with a switch)